### DISCUSSION TITLE
Download Problem

### DISCUSSION THREAD

**Discussion by zackmuc** [containrrr/watchtower#1138] [Q&A] [answered]
Hello I have some containers with portainer, and when i watch the log of WatchTower I can see this:

time="2021-11-15T13:57:32+01:00" level=warning msg="Reason: registry responded to head request with \"401 Unauthorized\", auth: \"Bearer realm=\\\"https://auth.docker.io/token\\\",service=\\\"registry.docker.io\\\",scope=\\\"repository:library/sha256:pull\\\",error=\\\"insufficient_scope\\\"\"" container=/monitoring-cadvisor image="sha256:ea1bec0f4e089c55580ba4330537c2e6a8306ec9e3ade37b105d828ca08865fd"

I added my github user with repo_pass and repo_user but it does not work. Can someone help?

Thx 

Peter

**Comment by piksel** [comment, marked as answer]
See https://github.com/containrrr/watchtower/issues/1050

**Comment by zackmuc** [comment]
> See #1050

But this does not describe my problem

**Reply by piksel** [reply]
> image="sha256:ea1bec0f4e089c55580ba4330537c2e6a8306ec9e3ade37b105d828ca08865fd"

You have a container without any image name. Hence why watchtower cannot update it. The information about what image to check for updates is not present in the container.
